# 客户端
uint8 task_type    # 请求任务标识: 1识别零食, 2识别食物框, 3识别咖啡机 4.识别杯子
uint8 camera_id  # 是否是右摄像头 1:右 0:左
---
# 服务器响应发送的数据
uint8 task_type
uint8 camera_id
uint8[] snack_type
float32[] XY_list